nconnect.go

```diff
@@ -72,12 +78,29 @@ func Run(opts *config.Opts) {
 
 	seed, err := hex.DecodeString(opts.Seed)
 	if err != nil {
-		log.Fatal(err)
+		return nil, err
 	}
 
 	account, err := nkn.NewAccount(seed)
 	if err != nil {
-		log.Fatal(err)
+		return nil, err
+	}
+
+	if opts.Address {
```

Comment:
Changing orders will cause problems. Address will not be saved if `opts.Address` or `opts.WalletAddress` is true

Reply:
Yes, moved after of persistConf.Save()